### PR TITLE
feat: gh800. batch unsubscribe from parameters.

### PR DESCRIPTION
**Pull request by SimonWoodRhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-IME-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-IME [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-IME-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

Added dialog for batch unsubscribing from parameters

### CODE REVIEW

File: EngineeringModel/ViewModels/ElementDefinitionBrowser/ElementDefinitionsBrowserViewModel.cs
Comment (samatrhea) [inline EngineeringModel/ViewModels/ElementDefinitionBrowser/ElementDefinitionsBrowserViewModel.cs:1055]: Please replace `creating` with `deleting`

**Review by samatrhea — CHANGES_REQUESTED** [review]
code looks good, one minor update requested. No need for re-review from my side once fixed

**Review by alexatrhea — APPROVED** [review]
approved pending @samatrhea comment

**Pull request merged**
